internal/proto/controller/api/resources/managedgroups/v1/managed_group.proto

```diff
@@ -91,3 +96,16 @@ message OidcManagedGroupAttributes {
     }
   ]; // @gotags: `class:"public"`
 }
+
+// Attributes associated only with ManagedGroups with type "ldap".
+message LdapManagedGroupAttributes {
+  // The list of groups that make up the ManagedGroup
+  repeated string group_names = 100 [
```

Comment:
Nit: the other messages in this file start at a tag number of 10, is there a good reason for this to be 100?

Reply:
I prefer to move towards 100 increments and I view this change no different than adding increments of 10 to a proto with no gaps between field numbers.